Update emacs to 26.1

asmeurer opened the pull request.

asmeurer committed:
Update emacs to 26.1

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

asmeurer committed:
Add gnutls dependency

asmeurer committed:
Apparently make check doesn't work on release tarballs now

asmeurer committed:
See if we can remove the manual binary replacement of the work dir on osx

asmeurer committed:
Install pkg-config in the build environment

asmeurer committed:
Remove manual binary replacement (it doesn't seem to be necessary any more)

asmeurer commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 3.1.6 and pinning 2018.07.01

asmeurer committed:
Run conda smithy update-cb3

List of changes done to the recipe:
Renamed build with host
Removing toolchain in favour of compiler()
Moving pkg-config from host to build
Removing pinnings for libxml2 to use values from conda_build_config.yaml
Removing pinnings for ncurses to use values from conda_build_config.yaml
Removing pinnings for jpeg to use values from conda_build_config.yaml
Removing pinnings for libpng to use values from conda_build_config.yaml
Removing pinnings for libtiff to use values from conda_build_config.yaml
Removing pinnings for giflib to use values from conda_build_config.yaml
Removing pinnings for freetype to use values from conda_build_config.yaml
Removing pinnings for zlib to use values from conda_build_config.yaml
Removing pinnings for python to use values from conda_build_config.yaml
Adding C compiler

jakirkham commented:
After running `update-cb3`, this needs a re-rendering. As that didn't happen would re-render and bump the build number. Likely the cause of and fix to issue ( https://github.com/conda-forge/emacs-feedstock/issues/24 ). Will also need to pull the old packages.

Will be out of the office for about a week starting later today. So cannot assist personally. Will ask @conda-forge/core to intervene (and verify that this is the correct course of action).

asmeurer commented:
> Will also need to pull the old packages.

You mean remove from anaconda.org?
